zerver/lib/onboarding.py

```diff
@@ -261,6 +261,11 @@ def send_welcome_bot_response(send_request: SendMessageRequest) -> None:
     )
 
 
+ONBOARDING_MESSAGES_COUNT = 12
```

Comment:
Why is this number required? It feels a bit fragile to me across changes in this onboarding code; what if we just marked all the messages welcome bot had sent?

Reply:
Right, we don't need this number actually. I missed the fact that these are the only stream messages sent by welcome bot.